packages/documentation/src/pages/getting-started/common-tasks/new-unit-test.mdx

```diff
@@ -30,35 +25,26 @@ For example, you might create a unit test file for each page in a form and then
       - 📂 components
         - MyComponent.unit.spec.jsx
 ```
-
 Mocha runs any *file_name*.unit.spec.js file located in the `/src` folder. This file is usually located in a test directory close to the code being tested.
-
 [Run the test](/getting-started/common-tasks/test) locally through npm script commands, during the Jenkins build (Unit), and after merging to master.
    
 ## Unit test conventions
-
 Use the following conventions when writing a unit test.
-
 ```js
 import React from 'react';
 import { expect } from 'chai';
-import { shallow } from 'enzyme';
+import { render, screen } from '@testing-library/react';
 import MyComponent from '../../components/MyComponent';
-
 describe('my-application', () => {
   describe('MyComponent', () => {
-    it('renders privacy act disclosure when state.show is true', () => {
-        const wrapper = shallow(<MyComponent />);
-        ...
-        expect(someCondition).to.be.true;
-
-        wrapper.unmount();
-    })
+    it('renders privacy act disclosure when "show" is true', () => {
+        render(<MyComponent show />);
+        expect(screen.getByRole('heading')).to.have.text('Privacy Act Disclosure');
+    });
   });
 });
 ```
-
-- Use `describe` to organize tests by application and feature. Two levels is adequate depending on the size of the application being tested.
+- Use `describe` to organize tests by application and feature. Two levels is adequate depending on the size of the application being tested. Try to avoid nesting deeper than two levels.
```

Comment:
```suggestion
- Use `describe` to organize tests by application and feature. Two levels should be adequate depending on the size of the application being tested. Try to avoid nesting deeper than two levels.
```

Reply:
[x] Use "should be" in place of "is" to provide more flexibility